repository-data/application/src/main/resources/hcm-content/content/documents/administration/publication-system/labels.yaml

```diff
@@ -27,6 +28,7 @@
     - labels.search-ghost-text
     - labels.upcoming-publication
     - labels.api
+    - labels.download-pdf
```

Comment:
Just a note on 'hcm-content' files, you will need to get someone to add this manually in the CMS.

Reply:
Sure, that should be fine